src/vue/pages/IncomingWithdrawalRequests.vue

```diff
@@ -0,0 +1,368 @@
+          </template>
+
+          <withdrawal-request-details
+            :request="selectedRequest"
+            :is-pending="isRequestRejected"
+            @approve="approveRequest"
+            @reject="rejectRequest"
+          />
+        </drawer>
+
+        <div class="app__table incoming-withdrawal-requests__table">
+          <table>
+            <thead>
+              <tr>
+                <!-- eslint-disable max-len -->
+                <th :title="'incoming-withdrawal-requests.requestor-header' | globalize">
+                  {{ 'incoming-withdrawal-requests.requestor-header' | globalize }}
+                </th>
+                <th :title="'incoming-withdrawal-requests.amount-header' | globalize">
+                  {{ 'incoming-withdrawal-requests.amount-header' | globalize }}
+                </th>
+                <th :title="'incoming-withdrawal-requests.request-state-header' | globalize">
+                  {{ 'incoming-withdrawal-requests.request-state-header' | globalize }}
+                </th>
+                <!-- eslint-enable max-len -->
+              </tr>
+            </thead>
+
+            <tbody>
+              <tr
+                v-for="(request, index) in requestsHistory"
+                :key="index"
+              >
+                <td :title="request.requestor">
+                  <email-getter :account-id="request.requestor" />
+                </td>
+
+                <td :title="request.amount | formatMoney">
+                  {{ request.amount | formatMoney }}
+                  {{ request.asset }}
+                </td>
+
+                <td
+                  v-if="request.isApproved"
+                  class="request-state request-state--approved"
+                  :title="'request-states.request-approved-state' | globalize"
+                >
+                  {{ 'request-states.approved-state' | globalize }}
+                </td>
+
+                <td
+                  v-if="request.isPending"
+                  class="request-state request-state--pending"
+                  :title="'request-states.request-pending-state' | globalize"
+                >
+                  {{ 'request-states.pending-state' | globalize }}
+                </td>
+
+                <td
+                  v-if="request.isRejected"
+                  class="request-state request-state--rejected"
+                  :title="'request-states.request-rejected-state' | globalize"
+                >
+                  {{ 'request-states.rejected-state' | globalize }}
+                </td>
+                <!-- eslint-disable max-len -->
+
+                <td
+                  v-if="request.isPermanentlyRejected"
+                  class="request-state request-state--permanently-rejected"
+                  :title="'request-states.request-permanently-rejected-state' | globalize"
+                >
+                  {{ 'request-states.permanently-rejected-state' | globalize }}
```

Comment:
Move the state statuses translations to the `incoming-withdrawal-requests` scope

Reply:
For every component, you need to create a standalone translation scope. For after

Reply:
resolved